README.md

```diff
@@ -141,7 +141,7 @@ To add this library to your project, update your `Cargo.toml` file with the foll
 
 ```
 [dependencies]
-blitzar = "0.2.0"
+blitzar = <version_number>
```

Comment:
Does this get expanded by cargo or crates.io? Or is this what it will look like in plain text?

Reply:
It will look like this: 
![image](https://github.com/spaceandtimelabs/blitzar-rs/assets/6509385/99a6612d-acba-4c46-965a-faf2cf5e7c3f)

I would love for this to get automatically updated. Do you know how I can do that? I know `https://img.shields.io/crates/v/blitzar.svg` gets a badge with the version number in it, but it doesn't work in a code box.